spec/unit/base/RequestClient.spec.js

```diff
@@ -284,13 +284,10 @@ describe("lastRequest defined, lastResponse undefined", function () {
 });
 
 describe("User specified CA bundle", function () {
-  let client;
   let options;
   beforeEach(function () {
     axios.create.mockReturnValue(createMockAxios(Promise.reject("failed")));
 
-    client = new RequestClient();
```

Comment:
Could we still keep this? `let client = new RequestClient()` instead of initializing it in every test case?